packages/plugins/contracts/foundry/Propose.sol

```diff
@@ -1,3 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
```

Comment:
What happens here if the user is trying to deploy with a lower Solidity version, e.g. 0.8.0?

Reply:
They just specify the compiler version they want in their contract files. I believe this only specifies the version to use for this specific file. Our current test contracts are actually already like this. They use various versions below 0.8.15, but `ChugSplash.sol` uses 0.8.15 and it doesn't seem like an issue.